front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/TextQuestion/Analysis/index.tsx

```diff
@@ -89,51 +93,69 @@ const Analysis = ({
 
   const showAnalysisInsights =
     relevantAnalysis && relevantAnalysis.attributes.show_insights;
+
   const hideAnalysisInsights =
     relevantAnalysis && !relevantAnalysis.attributes.show_insights;
 
+  const noInsights = !relevantAnalysis || insights?.data.length === 0;
+
+  const goToAnalysis = () => {
+    if (relevantAnalysis?.id) {
```

Comment:
Here I am making sure we navigate to the already existing analysis if there is one and if not, we create it.